tests/dummy/content/the-editor.md

```diff
@@ -55,16 +55,6 @@ or
 - Feed the cat
 - Come up with idea for next story
 
-
-## Adding quotes
```

Comment:
Can you leave this section in the markdown? The idea was that the blockquote styles are now being provided by the styleguide so we don't need to include them in our styles, but they should still work 🎉